components/src/core/Drawer/DrawerFooter/DrawerFooter.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
There is something in this file that is causing an issue with linting.  Linting takes 4x more time whenever this file is included.  I played around with this for some time & cannot figure out what the culprit is.  I suggest making an issue for this & temporarily adding this file to the eslint ignore list. (see commit)

Reply:
Issue created https://github.com/brightlayer-ui/react-component-library/issues/433